src/types/types.ts

```diff
@@ -17,18 +17,17 @@ export enum QuestionType {
 }
 
 export enum AnswerType {
-  LField = 'lField',
-  SField = 'sField',
+  LargeInput = 'largeInput',
+  SmallInput = 'smallInput',
   Dropdown = 'dropdown',
-  Radio = 'radio',
```

Comment:
There are radio questions in the intake form. Is there a reason this is removed?

Reply:
added